datalad/distribution/create.py

```diff
@@ -68,6 +68,8 @@ class Create(Interface):
     def __call__(
             loc=None, description=None, no_annex=False, annex_version=None,
             annex_backend='MD5E'):
+        if description and no_annex:
+            raise ValueError("Incompatiable arguments.")
```

Comment:
Also try to describe what is incompatible?  that description is intended to describe the annex repository, and not applicable to a regular git repo